CONTRIBUTING.rst

```diff
@@ -11,6 +11,13 @@ Setting up the environment
 4. Install `pre-commit <https://pre-commit.com/>`_
 5. Run ``pre-commit install`` to install pre-commit hooks
 
+.. tip::
+  Many modern IDEs like PyCharm or VS Code will enable the poetry-managed virtualenv that is created in step 2 for you automatically.
+  If you're not using such an IDE then you will need to manually activate the virtualenv yourself otherwise you will encounter *"Command not found"* errors.
```

Comment:
```suggestion
  If your IDE / editor does not offer this functionality, then you will need to manually activate the virtualenv yourself. Otherwise you may encounter errors or unexpected behaviour when trying to run the commands referenced within this document.
```

Making it a bit less specific.